src/main/java/tools/util/ToolConsole.java

```diff
@@ -0,0 +1,39 @@
+package tools.util;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+/**
+ * A collection of methods for use by support tools to write messages to the console.
+ */
+public final class ToolConsole {
```

Comment:
Should this be maybe called `ToolLogger` instead of `ToolConsole`? I somewhat expected this class to extend GenericConsole to fit with `ErrorConsole` and `PerformanceConsole`. The usage here though reminds me more of `ClientLogger`

Reply:
:+1: Agreed.  Will change in about an hour.

Reply:
00b6070